org/mozilla/jss/tests/TestBufferPRFD.java

```diff
@@ -140,15 +146,15 @@ public synchronized static void TestSSLHandshake(String database, String passwor
                 int error = PR.GetError();
 
                 if (error != PRErrors.WOULD_BLOCK_ERROR) {
-                    System.out.println("Unexpected error: " + error);
+                    System.out.println("Unexpected error: " + new String(PR.ErrorToName(error)) + " (" + error + ")");
```

Comment:
Just wondering, why not make `PR.ErrorToName()` return a `String` object directly? The orinal `PR_ErrorToName()` returns a `char*` which presumably is meant to be used as a string.

Same question about `PR.GetErrorText()`.

Reply:
Ah good point, I could do that. The issue is that `jss/util/jssutil.h` doesn't currently have methods for converting `char *` to `jstring` so I'd want to add that wrapper first.

Reply:
Addressed in #212.